modules/lib/src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -2038,17 +2037,19 @@ export class ContentWizardPanel
 
             this.getWizardHeader().resetBaseValues();
 
-            return content;
-        }).then((content: Content) => {
-            this.formState.setIsNew(false);
-            this.contentWizardStepForm.validate();
-            this.xDataWizardStepForms.validate(false, true);
-            this.displayValidationErrors(!this.isValid());
-
```

Comment:
You have moved this to `postUpdatePersistedItem` but you are not calling it...

Reply:
It is being called in base WizardPanel.ts

Reply:
Also I'd mention that post save/update logic is spread into multiple methods/then clauses - one of the candidates to refactor and to be moved in some single place if possible